### PR TITLE
[6628] don't block message sending when mailserver disconnected

### PR DESCRIPTION
**Pull request by yenda**
status: ready

### CODE REVIEW

**Review by dmitryn — APPROVED** [review]

File: src/status_im/ui/screens/chat/input/send_button.cljs [hunk 1]
```diff
@@ -14,22 +14,20 @@
        (animation/timing spin-value {:toValue  to-spin-value
                                      :duration 300})))))
 
-(defn sendable? [input-text network-status mailserver-connected?]
+(defn sendable? [input-text network-status]
```
Comment (cammellos) [inline src/status_im/ui/screens/chat/input/send_button.cljs:17]: Probably a good candidate for some tests

**Comment by dmitryn** [comment]
It probably fixes https://github.com/status-im/status-react/issues/6628

**Review by janherich — APPROVED** [review]

**Comment by lukaszfryc** [comment]
While testing, I was able to reproduce the original issue. And, looks like it's gone now as I could not repro it on Desktop while upgrading from nightly to this PR 👍

I'm trying on mobile now. I'll let you know soon if it works there as well.

**Comment by lukaszfryc** [comment]
@yenda all looks good

**Pull request merged**